sources/europe/fr/FR-IGN-Ortho.geojson

```diff
@@ -0,0 +1,336 @@
+{
+    "type": "Feature",
+    "properties": {
+        "attribution": {
+            "url": "http://openstreetmap.fr/bdortho",
+            "text": "BDOrtho IGN",
+            "required": true
+        },
+        "description": "French IGN aeral imagery",
+        "url": "http://proxy-ign.openstreetmap.fr/bdortho/{zoom}/{x}/{y}.jpg",
```

Comment:
I see. There's  also a ticket on the iD repository, apparently: https://github.com/openstreetmap/iD/issues/3197